client.go

```diff
@@ -1408,6 +1408,25 @@ func linkProperty(key string, value interface{}) LinkOption {
 	}
 }
 
+// LinkSourceName sets the name of the source queue
+func LinkSourceName(name string) LinkOption {
+	return func(l *link) error {
+		l.name = name
+		return nil
+	}
+}
+
+// LinkSourceCapabilities sets the source capabilities
+func LinkSourceCapabilities(capabilities ...symbol) LinkOption {
```

Comment:
I like to keep details like symbol vs string types internal to the lib. The capabilities here should be `...string` and converted to `symbol` inside the function. This will require a loop, but I think it makes for a nicer to use API.

Reply:
Fixed